magento/magento2
# Issues 6729, 6457, 7362: Configurable product price options provider

Fix for lowest price options provider to retrieve products with additional attributes

### Description
_LowestPriceOptionsProvider_ returns products without attributes which are used for price calculation (e.g. tax adjustment)

### Fixed Issues
1. magento/magento2#6729: Configurable product old price with taxes displayed wrong
2. magento/magento2#6457: Expired special_price is still shown for configurable products when no variant is selected
3. magento/magento2#7362: Special price vigency for configurable childs (simple products associated) doesn´t work

### Manual testing scenarios
1. Create configurable product with tax, price is defined for each sku (Stores > Configuration > Sales > Tax > Catalog Prices = Including Tax)
2. Create tax rule 25%
3. Apply this tax rule to the product
4. In tax settings set all values to display prices without tax (Stores > Configuration > Sales > Tax > Display Prices = Excluding Tax)
5. Check old configurable product price
